LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2018 1up GmbH
+Copyright (c) 2020 1up GmbH
```

Comment:
Shouldn't this be `2018–2020`? Or maybe just `Copyright (c) 1up GmbH` so you do not have to update the file every year.

Reply:
Removed in 1724c2f.